x-pack/plugins/observability_solution/observability_ai_assistant/server/routes/functions/route.ts

```diff
@@ -33,15 +33,15 @@ const getFunctionsRoute = createObservabilityAIAssistantServerRoute({
 
     const client = await service.getClient({ request });
 
-    const [functionClient, knowledgeBaseInstructions] = await Promise.all([
+    const [functionClient, userInstructions] = await Promise.all([
```

Comment:
nit: should we perhaps call them `storedUserInstructions`? Right now we have a type called `UserInstruction` which is used for both request instructions and stored instructions. We could also change that to something like `AssistantInstruction`, and then we have three flavors: default/system (system makes more sense in isolation but the term is overloaded), user and request.

Reply:
Sorry, this got auto-merged. I'll take a look at your suggestions in a bit